modules/System Admin/theme_manage_uninstallProcess.php

```diff
@@ -67,9 +67,9 @@
             }
 
             if ($orphaned != 'true') {
-                $URLDelete = $URLDelete.'&return=success0';
-            } else {
                 $URLDelete = $URLDelete.'&return=warning0';
+            } else {
+                $URLDelete = $URLDelete.'&return=success2';
```

Comment:
Has this change flipped the error state? 🤔 It looks like $orphaned being true is a bad thing, so $orphaned != 'true' would be the success state?